libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/NotificationBitmapLoader.kt

```diff
@@ -39,21 +39,22 @@ class NotificationBitmapLoader @Inject constructor(
     /**
      * Get icon of a room.
      * @param path mxc url
+     * @param imageLoader Coil image loader
      */
-    suspend fun getRoomBitmap(path: String?): Bitmap? {
+    suspend fun getRoomBitmap(path: String?, imageLoader: ImageLoader): Bitmap? {
         if (path == null) {
             return null
         }
-        return loadRoomBitmap(path)
+        return loadRoomBitmap(path, imageLoader)
     }
 
-    private suspend fun loadRoomBitmap(path: String): Bitmap? {
+    private suspend fun loadRoomBitmap(path: String, imageLoader: ImageLoader): Bitmap? {
         return try {
             val imageRequest = ImageRequest.Builder(context)
                 .data(MediaRequestData(MediaSource(path), MediaRequestData.Kind.Thumbnail(1024)))
                 .transformations(CircleCropTransformation())
                 .build()
-            val result = context.imageLoader.execute(imageRequest)
+            val result = imageLoader.execute(imageRequest)
```

Comment:
That was the bug, the default `imageLoader` is not correct when there is a session, but the UI is dead (spotted by @ganfra )